### PR TITLE
Bring deployment to the repo

### PR DESCRIPTION
**Pull request by enolfc**
# Summary

Bring deployment of the VM dealing with cloud-info-provider to this repo

---

**Related issue :**

### CODE REVIEW

File: deploy/CESNET-MCC.tfvars [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Network
+net_id = "5b8fbb48-461d-4907-ac79-7f7b2ebd17f9"
+
+# Flavor: standard.medium   2cores/4GB RAM
+flavor_id = "4c153ce3-a163-4668-baa7-2cbcb57e2dd8"
+
+# Image: ubuntu bionic
+image_id = "e8d75fc1-ac32-4851-90b5-b4c925e9e6f8"
+
```
Comment (gwarf) [inline deploy/CESNET-MCC.tfvars:9]:
```suggestion
```

File: deploy/README.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+This directory manages the deployment automatically on selected providers of
+the infrastructure
+
+## Deployment
+
+Deployment is performed in 2 phases:
+
+1. Terraforming the VM where the cloud-info-provider is run
+
```
Comment (gwarf) [inline deploy/README.md:11]:
```suggestion
```

File: deploy/cloud-init.yaml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+packages:
+- git
+- ansible
+- jq
+
+write_files:
+- content: |
+    #!/bin/sh
+
```
Comment (gwarf) [inline deploy/cloud-init.yaml:28]: Should we run with things like `set -e`?

**Review by gwarf — COMMENTED** [review]
Added a few dumb suggestions and a comment, please check.

File: deploy/cloud-init.yaml (hunk 3 above)
Comment (enolfc) [reply deploy/cloud-init.yaml:28]: Yes. Needs a bit of better error handling yet. I will add something

**Review by gwarf — APPROVED** [review]
LGTM

**Pull request merged**